### DISCUSSION TITLE
allow custom colors for menu-button-background.png

### DISCUSSION THREAD

**Discussion by GRBL-F** [ppy/osu#21741] [Ideas] [answered]
TLDR; Allow custom color for the menu-button-background.png skin element, probably dictated by rgb values in the skin.ini file.

As of now the only real color option for menu-button-background is blue. when i search for songs by the artist, for example, I look for the group that's lighter than all the rest. unfortunately, right now this group is only blue and sometimes hard to distinguish from the others. this also limits or clashes with color schemes you may be trying to achieve. for example a black and white color scheme is impossible if you want to be able to distinguish what category your song is in. it'd be nice to be able to change this color, possibly by adding something like "MenuButtonBGColor 255, 255, 255" or "MenuButtonBGColorOver 255, 255, 255" in the skin.ini file. allowing you to be able to change the color of the base element as well as when it's hovered over or lighter to distinguish the song is in that category.

**Comment by nagi-desuuu** [comment, marked as answer]
1. This is an osu!stable issue. This repo is for osu!lazer only.
2. stable is feature-locked so your request won't be implemented anytime soon.
